src/main/config/config.c

```diff
@@ -136,6 +136,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
 
     pidProfile->acc_soft_lpf_hz = 15;
     pidProfile->gyro_soft_lpf_hz = 60;
+	pidProfile->gyro_soft_notch_cutoff_hz = 130;
+	pidProfile->gyro_soft_notch_hz = 260;
```

Comment:
I think for INAV defaults might be more conservative (or notch can simply be disabled). INAV it's targeted also for a bigger quads and airplanes.